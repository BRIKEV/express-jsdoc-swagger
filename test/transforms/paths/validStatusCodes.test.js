const validStatusCodes = require('../../../transforms/paths/validStatusCodes');

test('Valid status codes snapshot', () => {
  expect(validStatusCodes).toMatchSnapshot();
});
