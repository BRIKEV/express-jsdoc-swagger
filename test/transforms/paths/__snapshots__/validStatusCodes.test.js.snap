// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Valid status codes snapshot 1`] = `
Object {
  "100": "Continue",
  "101": "Switching Protocols",
  "102": "Processing",
  "1XX": "1XX Range",
  "200": "OK",
  "201": "Created",
  "202": "Accepted",
  "203": "Non Authoritative Information",
  "204": "No Content",
  "205": "Reset Content",
  "206": "Partial Content",
  "207": "Multi-Status",
  "2XX": "2XX Range",
  "300": "Multiple Choices",
  "301": "Moved Permanently",
  "302": "Moved Temporarily",
  "303": "See Other",
  "304": "Not Modified",
  "305": "Use Proxy",
  "307": "Temporary Redirect",
  "308": "Permanent Redirect",
  "3XX": "3XX Range",
  "400": "Bad Request",
  "401": "Unauthorized",
  "402": "Payment Required",
  "403": "Forbidden",
  "404": "Not Found",
  "405": "Method Not Allowed",
  "406": "Not Acceptable",
  "407": "Proxy Authentication Required",
  "408": "Request Timeout",
  "409": "Conflict",
  "410": "Gone",
  "411": "Length Required",
  "412": "Precondition Failed",
  "413": "Request Entity Too Large",
  "414": "Request-URI Too Long",
  "415": "Unsupported Media Type",
  "416": "Requested Range Not Satisfiable",
  "417": "Expectation Failed",
  "418": "I'm a teapot",
  "419": "Insufficient Space on Resource",
  "420": "Method Failure",
  "422": "Unprocessable Entity",
  "423": "Locked",
  "424": "Failed Dependency",
  "428": "Precondition Required",
  "429": "Too Many Requests",
  "431": "Request Header Fields Too Large",
  "4XX": "4XX Range",
  "500": "Server Error",
  "501": "Not Implemented",
  "502": "Bad Gateway",
  "503": "Service Unavailable",
  "504": "Gateway Timeout",
  "505": "HTTP Version Not Supported",
  "507": "Insufficient Storage",
  "511": "Network Authentication Required",
  "5XX": "5XX Range",
  "default": "Default response",
}
`;
